RecoBTag/CTagging/python/charmTaggerProducer_cfi.py

```diff
@@ -32,3 +37,41 @@
    variables = c_vs_b_vars_vpset
    )
 
+#
+# Negative tagger
```

Comment:
cfi files aren't really meant for definitions of multiple modules.
It should technically be just one and a cfi the file name better match the name of the module or PSet instance defined inside it.
Please move the various definitions to charmTaggerProducer_cff